builtin/game/falling.lua

```diff
@@ -122,6 +123,24 @@ core.register_entity(":__builtin:falling_node", {
 			})
 		end
 
+		-- Set collision box (certain nodeboxes only for now)
+		local nb_types = {fixed=true, leveled=true, connected=true}
+		if def.drawtype == "nodebox" and def.node_box and
+			nb_types[def.node_box.type] then
+			local box = table.copy(def.node_box.fixed)
+			if type(box[1]) == "table" then
+				box = #box == 1 and box[1] or nil -- We can only use a single box
+			end
+			if box then
+				if def.paramtype2 == "leveled" and (self.node.level or 0) > 0 then
+					box[5] = -0.5 + self.node.level / 64
```

Comment:
Shouldn't this now be `self.node.level / def.leveled_max` plus another check above?

Reply:
https://github.com/minetest/minetest/blob/7d3972a5049324f776ab008894c34569641f0073/src/mapnode.cpp#L248-L249